tasks/push-sbom-to-pyxis/README.md

```diff
@@ -11,6 +11,9 @@ Tekton task that extracts sboms from pull specs and pushes them to Pyxis.
 | pyxisSecret | The kubernetes secret to use to authenticate to Pyxis. It needs to contain two keys: key and cert | No | - |
 | server | The server type to use. Options are 'production' and 'stage' | Yes | production |
 
+## Chnages since 0.3.1
```

Comment:
typo

Reply:
I will never understand how I can continually misspell "changes"
Should be fixed now